developer_manual/core/unit-testing.rst

```diff
@@ -253,3 +253,49 @@ Here are some useful links about how to write unit tests with Jasmine and Sinon:
    
 .. _the PHPUnit documentation: https://phpunit.de/manual/current/en/installation.html
 .. _the writing tests section: http://www.phpunit.de/manual/current/en/writing-tests-for-phpunit.html
+
+
+UI testing in Core with selenium
+--------------------------------
+
+Requirements:
+~~~~~~~~~~~~~
+- ownCloud >= 10.0 (make sure you have an running instance of ownCloud complete setup)
+- admin user called "admin" with the password "admin"
+- no self-signed SSL certificates
+- testing utils (running ``make`` in your terminal from the ``webroot`` directory will install them)
+- Selenium standalone server (download Selenium standalone server jar from http://docs.seleniumhq.org/download/)
+- browser installed that you would like to test on
+- webdriver for the browsers you want to test (e.g. download the chromedriver from: https://sites.google.com/a/chromium.org/chromedriver/ , firefox webdriver is included in the selenium server)
+
+Set Up test
+~~~~~~~~~~~
+- place the selenium standalone server jar file and the webdriver(s) somewhere in the same folder
+- start the selenium server ``java -jar selenium-server-standalone-3.0.1.jar -port 4445``
+- set the following environment variables:
+  - SRV_HOST_NAME (the hostname where ownCloud runs)
```

Comment:
I'd suggest surrounding SRV_HOST_NAME, and the other three environment variables in double backticks